modules/stanford_profile_styles/lib/scss/wysiwyg.scss

```diff
@@ -1,5 +1,5 @@
 @charset 'UTF-8';
-@import './config';
+@import 'config';
```

Comment:
Why remove the path?

Reply:
phpstorm likes to refactor all the things.

Reply:
sass is funny with imports. It will first look for a relative file matching the import statement and if it does not find a match it will start looking at all the sass include paths which includes the npm directory. So if in the future the config file is renamed or moved it is possible that this import statement could pick up another config file but I guess that would be the same ./ or not. FYI at his point.